build-tools/src/main/resources/software/amazon/awssdk/spotbugs-suppressions.xml

```diff
@@ -359,4 +359,8 @@
         <Class name="software.amazon.awssdk.v2migration.EnumCasingToV2$Visitor"/>
         <Bug pattern="NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE"/>
     </Match>
+    <Match>
+        <Class name="software.amazon.awssdk.utils.uri.SdkUri" />
+        <Bug pattern="BC_UNCONFIRMED_CAST_OF_RETURN_VALUE" />
```

Comment:
Question: where do we cast?

Reply:
here: https://github.com/aws/aws-sdk-java-v2/pull/6087/files#diff-ba4e76ea7c37f0a322912c476915fa2de501d6ff5f3af283be7f54babf846f00R81-R88